debugging/book-library/script.js

```diff
@@ -1,103 +1,101 @@
 let myLibrary = [];
 
-window.addEventListener("load", function (e) {
+window.addEventListener("load", function () {
   populateStorage();
-  render();
-});
 
-function populateStorage() {
-  if (myLibrary.length == 0) {
-    let book1 = new Book("Robison Crusoe", "Daniel Defoe", "252", true);
-    let book2 = new Book(
-      "The Old Man and the Sea",
-      "Ernest Hemingway",
-      "127",
-      true
-    );
-    myLibrary.push(book1);
-    myLibrary.push(book2);
-    render();
+  function populateStorage() {
+    if (myLibrary.length === 0) {
+      let book1 = new Book("Robison Crusoe", "Daniel Defoe", "252", true);
+      let book2 = new Book(
+        "The Old Man and the Sea",
+        "Ernest Hemingway",
+        "127",
+        true
+      );
+      myLibrary.push(book1);
+      myLibrary.push(book2);
+      render();
+    }
   }
-}
 
-const title = document.getElementById("title");
-const author = document.getElementById("author");
-const pages = document.getElementById("pages");
-const check = document.getElementById("check");
+  const title = document.getElementById("title");
+  const author = document.getElementById("author");
+  const pages = document.getElementById("pages");
+  const check = document.getElementById("check");
+  const submitButton = document.querySelector('[data-atr="submit"]');
 
-//check the right input from forms and if its ok -> add the new book (object in array)
-//via Book function and start render function
-function submit() {
-  if (
-    title.value == null ||
-    title.value == "" ||
-    pages.value == null ||
-    pages.value == ""
-  ) {
-    alert("Please fill all fields!");
-    return false;
-  } else {
-    let book = new Book(title.value, title.value, pages.value, check.checked);
-    library.push(book);
-    render();
+  function submit() {
+    if (
+      title.value.trim() === "" ||
+      pages.value.trim() === "" ||
+      isNaN(pages.value) ||
+      parseInt(pages.value) <= 0 ||
+      !Number.isInteger(pages.value)
+    ) {
+      alert("Please fill all fields appropriate!");
+    } else {
+      let book = new Book(
+        title.value,
+        author.value,
+        pages.value,
+        check.checked
+      );
```

Comment:
Using input values directly without first sanitise them is dangerous.

What if the user enters "                                      foo" as title, or "000000007" as pages?

And if you have time for more challenges, what if the user enters 
"</table> haha" or "Hello &copy" as title?